config/menus/maps.cfg

```diff
@@ -776,13 +776,57 @@ newgui maps [ ui_stay_open [ octapaks [
     ui_strut 0.5
     ui_font super [
         ui_center [
+            show_force_button = 0
+            if (&& (getclientpriv $getclientnum $vetolock) (=s (getmastermode 1) "veto")) [
+                show_force_button = 1
+            ]
+
+
             if (&& (>= $chosenmode 0) (stringlen $chosenmap)) [
-                ui_button (? (isonline) "^fgVote" "^fgPlay")  [if (isonline) [showgui maps 2]; (? (= (gamemode) 1) (savewarnchk [start $chosenmap $chosenmode (chosenmuts)]) (start $chosenmap $chosenmode (chosenmuts))); mapsearch = ""]
+                ui_button (? (isonline) "^fgVote" "^fgPlay") [
+                    if (isonline) [
+                        showgui maps 2
+                    ]
+                    if (= (gamemode) 1) [
+                        savewarnchk [ start $chosenmap $chosenmode (chosenmuts)]
+                    ] [
+                        start $chosenmap $chosenmode (chosenmuts)
+                    ]
+                    mapsearch = ""
+                ]
+
+                if $show_force_button [
+                    ui_strut 1.5
+                    ui_button "^fzwyForce" [
+                        if (isonline) [
+                            showgui maps 2
+                        ]
+                        if (= (gamemode 1) [
+                            savewarnchk [ mapsexec $chosenmap $chosenmode (chosenmuts) ]
+                        ] [
+                            mapsexec $chosenmap $chosenmode (chosenmuts)
+                        ]
+                        mapsearch = ""
+                    ]
+                ]
             ] [
                 ui_text (? (isonline) "^fdVote" "^fdPlay")
+
+                if $show_force_button [
+                    ui_strut 1.5
+                    ui_text "^fdForce"
+                ]
             ]
         ]
     ]
+    ui_font "little" [ ui_center [
+        if (&& (isonline) (getclientpriv $getclientnum $masterlock) (!=s (getmastermode 1) "veto")) [
+            ui_button (format "Change mastermode from ^fg%1 ^fwto ^fzwyveto" (getmastermode 1)) [
+                mastermode (! (getmastermode))
+            ]
+        ]
+    ] ]
```

Comment:
This is slightly small or not spaced enough, it should be more distinguished from the button above it.

Reply:
huh, yeah I agree, I'll redesign that part